XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs

```diff
@@ -0,0 +1,33 @@
+﻿using XamarinCommunityToolkit.Extensions;
+using System.Globalization;
+using Xamarin.Forms;
+using System;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Converts/Extracts the incoming value from ItemTappedEventArgs object and returns the value of Item property from it.
+    /// </summary>
+    public class ItemTappedEventArgsConverter : ValueConverterExtension, IValueConverter
+    {
+        /// <summary>
+        /// Converts/Extracts the incoming value from ItemTappedEventArgs object and returns the value of Item property from it.
+        /// </summary>
+        /// <param name="value">The value to convert.</param>
+        /// <param name="targetType">The type of the binding target property.</param>
+        /// <param name="parameter">Additional parameter for the converter to handle. Not implemented.</param>
+        /// <param name="culture">The culture to use in the converter.</param>
+        /// <returns>A Item object from object of type ItemTappedEventArgs.</returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (!(value is ItemTappedEventArgs itemTappedEventArgs))
+            {
+                throw new ArgumentException("Expected value to be of type ItemTappedEventArgs", nameof(value));
+            }
+
+            return itemTappedEventArgs.Item;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) => throw new NotImplementedException();
```

Comment:
```suggestion
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
            => value is ItemTappedEventArgs itemTappedEventArgs
                ? itemTappedEventArgs.Item
                : throw new ArgumentException("Expected value to be of type ItemTappedEventArgs", nameof(value));
                
        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) 
            => throw new NotImplementedException();
```

Reply:
Ok, makes sense I have noticed a similar coding style in other converters. Thanks!

Reply:
1) Usually, we avoid braces in **IF/ELSE/WHILE etc.** statements if there is only one line under them.
2) Also **=>** in methods should be started from the new line.

Reply:
I will keep that in mind for the next contributions :) Thank you!